package.json

```diff
@@ -42,7 +41,8 @@
   },
   "peerDependencies": {
     "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react-dom": "^0.14.0",
+    "react-tap-event-plugin": "^0.2.0"
```

Comment:
The react tap event needs to be handle by the application that is using material-ui. Hence, it can't be a dependency. It needs to be a peer dependancy.